packages/radio-group/src/RadioGroup.story.tsx

```diff
@@ -25,22 +25,11 @@ const meta: StoryMetaType<typeof RadioGroup> = {
     },
   },
   args: {
-    children: (
-      <>
-        <Radio checked value="1">
-          Radio Input 1
-        </Radio>
-        <Radio default value="2" description="This is a description">
-          Radio Input 2
-        </Radio>
-        <Radio disabled value="Selection-4">
-          Disabled Option
-        </Radio>
-      </>
-    ),
+    children: <></>,
```

Comment:
This didn't work previously, because RadioGroup depends on logic where we do not recursively iterate through children, we only check children at the first level.